tests/subsys/llext/simple/src/test_llext_simple.c

```diff
@@ -176,7 +167,7 @@ void load_call_unload(const struct llext_test *test_case)
 	k_thread_join(&llext_thread, K_FOREVER);
 
 	if (test_case->test_cleanup) {
-		test_case->test_cleanup(&ext);
+		test_case->test_cleanup(ext);
```

Comment:
merge into previous one

Reply:
Botched the fixups! Thanks for spotting this :+1: